lib/DonorsList/DonorsContainer.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FieldArray } from 'react-final-form-arrays';
+
+import {
+  Col,
+  Loading,
+  Row,
+} from '@folio/stripes/components';
+
+import DonorsList from './DonorsList';
+import { useFetchDonors } from './hooks';
+
+function DonorsContainer({ name, donorOrganizationIds }) {
+  const [donorIds, setDonorIds] = useState(() => donorOrganizationIds);
```

Comment:
Why is lazy initialization needed here?

Reply:
Good point. We can remove it. Initial implementation I had a function to get the ids.